docs/source/concepts/deploy.rst

```diff
@@ -6,15 +6,73 @@ Deploying Bento
 Deployment Overview
 -------------------
 
-The three most common deployment options with BentoML are:
-
-- 🐳 Generate container images from Bento for custom docker deployment
-- `🦄️ Yatai <https://github.com/bentoml/Yatai>`_: Model Deployment at scale on Kubernetes
-- `🚀 bentoctl <https://github.com/bentoml/bentoctl>`_: Fast model deployment on any cloud platform
-
-
-Containerize Bentos
--------------------
+Here are the four most common deployment options with BentoML:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Deploy Method 
+     - Pros ❤️ 
+     - Cons 💔 
+   * - `🍱 BentoCloud <https://www.bentoml.com/>`_
+     - Serverless cloud for AI, fully managed, made for BentoML
+     - Limited availability, `sign up here <https://www.bentoml.com/bento-cloud/>`_ today for early access
+   * - Docker Containers 
+     - Flexible for deploying almost anywhere, easy to get started 
+     - Requires DevOps work for production deployment, lacks advanced scaling capabilities
+   * - `Yatai on Kubernetes <https://github.com/bentoml/Yatai>`_
+     - Cloud-native AI deployment on Kubernetes, comes with advanced auto-scaling and CI/CD workflows
+     - Requires professional DevOps team to maintain and operate
+   * - Cloud Deployment with `BentoCTL <https://github.com/bentoml/bentoctl>`_ 
+     - Great for proof-of-concept deployments directly running on public cloud services (EC2, ECS, SageMaker, Lambda, GCP, etc) 
+     - Requires working knowledge of Cloud Services and their limitations for AI-specific workloads
+
+
+
+Feature comparison across deployment options:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Feature
+     - `🍱 BentoCloud <https://www.bentoml.com/>`_
+     - Docker Containers 
+     - `Yatai on Kubernetes <https://github.com/bentoml/Yatai>`_
```

Comment:
Re-arrange Docker Containers and Yatai since Yatai looks like the 2nd best option?